third-party/traefik/package.json

```
@@ -0,0 +1,67 @@
+{
+  "name": "@kubernetes-models/traefik",
+  "version": "2.3.0",
```

Changeset will bump the version automatically.

```suggestion
  "version": "0.0.0",
```

---

Sorry for the delay, I've made the changes you requested.